build.gradle

```diff
@@ -848,6 +848,11 @@ project('test:system') {
 
     task startK8SystemTests(type: Test) {
         tasks.withType(Exec) { environment "tier2Type", System.getProperty("tier2Type", "nfs") }
+        tasks.withType(Exec) { environment "imageVersionPrOp",System.getProperty("imageVersionPrOp", "latest") }
```

Comment:
Can we have a better naming here? As I understand this is the Pravega operator version.
Perhaps `pravegaOperatorVersion` or something still better.
Let's follow the same naming convention across all the properties.

Reply:
Changed all 3 operator names to pravegaOperatorVersion, bookkeeperOperatorVersion, zookeeperOperatorVersion